sdk/devcenter/azure-developer-devcenter/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0 (2023-05-23)
 
-### Features Added
-
-### Breaking Changes
```

Comment:
After PR finalize and before it merge, you need to list what's been broken since last preview.